tests/protractor/page-objects/forms/delivery-report.po.js

```diff
@@ -1218,21 +1188,27 @@ module.exports = {
 
   //note to CHW
   getNoteToCHW: () => {
-    return element(by.name('/delivery/group_note/g_chw_sms')).getText();
+    return element(by.css('textarea')).getAttribute('value');
```

Comment:
I remember struggling to get the text from the textarea (`getText()` did not work)and after some attempts, I looked at the console and settled with  `getAttribute('value')`.  Indeed, looking at the html `<textarea name="/delivery/group_note/g_chw_sms" data-constraint="string-length(.) <= 715" data-calculate="jr:choice-name( /delivery/group_note/default_chw_sms ,' /delivery/group_note/default_chw_sms ')" data-type-xml="string"></textarea>` there seems to be no text in that textarea. And looking at the 'properties' , innerText and innerHtml are both empty: 
![image](https://user-images.githubusercontent.com/6979995/27269112-3d4cb1b0-5508-11e7-8744-a27ff0e6f7e0.png)
So, the actual next is under 'value': 
![image](https://user-images.githubusercontent.com/6979995/27269139-69b96806-5508-11e7-95f2-c330268233cd.png)